scripts/conventional-changelog/index.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+var parserOpts = {
```

Comment:
Oh.. So you are reading back the commit history and generating the changelog out of it somehow?

On some projects what I did is that if the CHANGELOG + state of the project is mixed of good commits and bad commits then I'll configure conventional-changelog to read the **new** commits well and only append to the CHANGELOG file (that I'll rename to CHANGELOG.md). And that's it, we can keep this script but I was a bit surprised to see it (seemed redundant with conventional-changelog defaults)